From 64baa7c5052f1dbbd156932552d1166b5c1d40ae Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Mon, 14 Sep 2009 08:41:55 +0000
Subject: [PATCH] abuild-sign: set permissions on signed index to 644

mktemp set it to 600 so we need to manually set it to 644
---
 abuild-sign.in |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/abuild-sign.in b/abuild-sign.in
index 2aa525e..86b3b15 100644
--- a/abuild-sign.in
+++ b/abuild-sign.in
@@ -80,6 +80,7 @@ for f in "$@"; do
 	cat "$tmptargz" "$i" > "$tmpsigned"
 	rm -f "$tmptargz"
 	mv "$tmpsigned" "$i"
+	chmod 644 "$i"
 	if [ -z "$quiet" ]; then
 		echo "Signed $i"
 	fi
-- 
1.6.4.2

